football_dictionaries/assignment_3.py

```diff
@@ -1,2 +1,186 @@
-def players_by_country_and_position(squads_list):
-    pass
+import pprint
+#from squads_data import SQUADS_DATA  
+from assignment_1 import players_as_dictionaries
```

Comment:
This line is failing under python 3.x because 3.x expects relative imports (imports from within the same package/directory) to be *explicit* you can remedy this by changing it to:
`from .assignment_1 import players_as_dictionaries`
Note the `.` in front of `assignment_1` this explicitly states the file you're importing from is in the same directory.

Reply:
ok, will modify that. thank you!